### ISSUE TITLE
Recast animation jumpy

### ISSUE THREAD

**Issue by GetAwayCoxn** [ThornyFFXI/tTimers#18]
Ive been messing about with a test account and noticed a quirk with the recast animation. Hard to describe but was able to catch it on video (shitty phone video sorry), curious to know if anything can/should be done to prevent the timers jumping back and forth when the time remaining is so close.

https://github.com/ThornyFFXI/tTimers/assets/66495755/6ee654de-f441-4de0-b0fa-7bb56fee7656

**Comment by ThornyFFXI** [comment]
It has pretty decent precision (1/60 sec), so that should be an extremely rare case.  I'm not sure why it'd jump back and forth, but it's sourcing directly from the client so that would come down to how the client is representing the times.  I'll have to look into it, but I don't think it's likely to be a widespread issue.

**Comment by GetAwayCoxn** [comment]
Fair enough, I wanted to at least bring it to your attention and hear your thoughts. I do not have any accurate data but I can say that in about 4 hours last night testing other non related stuff I noticed it exactly 6 times, so rare but not that rare? It was mostly showing up when using Fight (short recast and spammed a lot more of course) and it finally happened with ja's with long enough recasts that I could get some evidence of it. I would think that if this was happening a lot though that someone would have mentioned it on discord at least, so as you said, probably not widespread issue.
